.changeset/plenty-clouds-guess.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend': minor
```

Comment:
Think that this should be a patch change as the `auth-backend` is <1.x

Reply:
Good call!